app/src/main/res/values-da/strings.xml

```diff
@@ -108,7 +108,7 @@
 
     <!-- This string is used as the title of the pop-ups dialog used to remove sites from the currently
          allowed pop-up sites. -->
-    <string name="settings_privacy_policy_popups_title">Pop op-tilladelser</string>
+    <string name="settings_privacy_policy_popups_title_v1">Pop op-tilladelser</string>
```

Comment:
As a general rule, you shouldn't modify localized files. But this is definitely wrong:
- You're adding a new string in en-US, `settings_privacy_policy_popups_title_v1` with text `Exceptions for pop-ups`. The old one was `settings_privacy_policy_popups_title`, `Pop-ups permissions`.
- By doing these changes, all locales will have this new string "pre-translated", but with the wrong reference (the old one, `Pop-ups permissions`.

Please revert all changes to the localized files, only add `settings_privacy_policy_popups_title_v1` to the English/source file as you already did.

Reply:
Thanks, fixed.